src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/CollectionBuilderAttribute.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.CompilerServices
+{
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Interface, Inherited = false, AllowMultiple = true)]
```

Comment:
Perhaps I have missed something but this differs from the one mentioned in [the proposal](https://github.com/dotnet/csharplang/blob/main/proposals/collection-literals.md#create-methods). Specifically the proposal calls for `AllowMultiple = false`.